src/Components/Homepage.css

```diff
@@ -80,49 +83,88 @@ body {
     width: 1200px;
 }
 
-.card {
-    height: 458px;
-    background-color: #555555;
-    margin-bottom: 22px;
+.add-modal {
+    left: 0;
+    top: 0;
+    width: 100%;
+    height: 100%;
+    overflow: auto;
+    background-color: rgba(0, 0, 0, .4);
+    position: fixed;
+    backdrop-filter: blur(8px);
+    padding-top: 128px;
```

Comment:
Not an error, but using pixels for styling can be weird sometimes. It can be hard to fix and hard to read

Reply:
fixed